From a640c5e357ce2aafb6edcba9b6c641cbc7753880 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Wed, 24 Mar 2021 20:57:44 -0400
Subject: [PATCH 1/1] doc/info/build_html.sh.in: use @PYTHON@ for the python
 interpreter.

Maxima's ./configure script already detects (or is prodded to use) a
certain python interpreter. The build_html.sh script, however, is
still using a hard-coded "python" command. On many systems, "python"
will be different from the python interpreter that we want to use; for
example when "python" is python-2.7.x and "python3" is what we want to
use with Maxima.

This commit tweaks doc/info/build_html.sh.in to use the value of the
$PYTHON variable instead.

Maxima-Bug: https://sourceforge.net/p/maxima/bugs/3754
Gentoo-Bug: https://bugs.gentoo.org/766291
---
 doc/info/build_html.sh.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/info/build_html.sh.in b/doc/info/build_html.sh.in
index 37930e9..c10e38d 100755
--- a/doc/info/build_html.sh.in
+++ b/doc/info/build_html.sh.in
@@ -46,7 +46,7 @@ do
   echo "filenamebase = \"$filenamebase\""
   cat $f
 done | @AWK@ '!/^@c / && !/^@c$/ && (/^@deffn/ || /^@defvr/ || /^@end deffn/ || /^@end defvr/ || /@category/ || /@node/ || /^filenamebase/)' | @SED@ -f @abs_srcdir@/extract_categories1.sed | @AWK@ -f @abs_srcdir@/extract_categories1.awk > make-categories.py 
-python make-categories.py 
+@PYTHON@ make-categories.py 
 @SED@ -e 's/^@bye/@node Documentation Categories, , Function and Variable Index\n@chapter Documentation Categories/' @abs_srcdir@/maxima.texi > maxima.texi 
 ( for f in Category-*.texi; do echo '@include' $f; done ; echo @bye ) >> maxima.texi 
 
-- 
2.26.2

